ush/python/pygfs/utils/marine_da_utils.py

```diff
@@ -0,0 +1,105 @@
+import f90nml
```

Comment:
Please use `type hints` to provide input/output types in the functions/classes.
While dynamic typing offers flexibility, it also creates room for potential bugs.
type hinting can significantly enhance code readability, help with documenting and debugging to ensure the right datatypes are being passed.

Reply:
Ok. Will push the changes after testing.